src/src/auths/auth-spa.c

```diff
@@ -1406,7 +1406,7 @@ spa_build_auth_challenge (SPAAuthRequest * request, SPAAuthChallenge * challenge
   int p = (int)getpid();
   int random_seed = (int)time(NULL) ^ ((p << 16) | p);
 
-  request = request;  /* Added by PH to stop compilers whinging */
+  request = request;  /* Added by PH to stop compilers whining */
```

Comment:
I'm aware that `whinging` is a correct word in en-GB, but it's ~5% as popular as the en-US spelling. 
Technically this is an en-US commit, since the en-GB word isn't misspelled, but...